libs/shared/ui/src/lib/components/icon/icons-git/bitbucket.tsx

```diff
@@ -0,0 +1,41 @@
+import { IconProps } from '../icon'
+
+export function BitbucketIcon(props: IconProps) {
+  return (
+    <svg
+      {...props}
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
can't we just have some kind of svg loader to load icons from an assets folder instead of creating a component for each icon ?

Reply:
After having used a lot the svg loader we have in our angular project, I don't see finally to much the difference and the real simplification that it provides over this method with one component by icon. 
This here also allows us a total control over any icon which I kind of appreciate. While digging on a way to handle icons with react, I did not find any convincing solution and this kind of solution was the one coming most often and the cleanest in my opinion.
In any case for now, it's just for the guideline icons because for the rest we're gonna use the font-awesome font.